andela/healthchecks-shockwave
# 150627367 team access scope

#### What does this PR do?
Assign checks to a team member when inviting them.
After an invite, a team member can only see checks that they have been assigned.
Only a team member and team owner can see checks that the team member has added.

#### Description of Task to be completed?
Add a list of the team owner's checks in the invite to team modal.
Display on team dashboard only the checks that the team owner invited the team member to monitor.

#### How should this be manually tested?
- After cloning the repo and installing requirements, cd into it and run `python manage.py test` in the virtual environment.
- To use the feature, run `python manage.py runserver`. \
Use `admin` endpoint to access Django administrator GUI and navigate to the profile for the superuser. Turn on `Team Access Allowed`. \
Go to the site using the button 'View Site' and add a few checks. \ 
Navigate to the endpoint `/accounts/profile`. Here, click on 'Invite' to invite a user by entering their email address and select the job checks they can access by using the checkboxes next to the respective job check. If the table with the job checks exceeds the maximum height, the user can scroll through the table. \
Use the log in url sent to the terminal to log in with the new account. Dashboard automatically displays the previously selected checks.

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/150627367
